cmd/k3s/main.go

```diff
@@ -79,11 +80,15 @@ func findDataDir() string {
 // binaries, it calls it directly and returns true. If it's not an external binary,
 // it returns false so that standard CLI wrapping can occur.
 func runCLIs(dataDir string) bool {
-	progName := filepath.Base(os.Args[0])
-	switch progName {
-	case "crictl", "ctr", "kubectl":
-		if err := externalCLI(progName, dataDir, os.Args[1:]); err != nil {
-			logrus.Fatal(err)
+	if os.Getenv("CRI_CONFIG_FILE") == "" {
```

Comment:
This appears to be undoing a change I just made, can you fix it so that you're just adding etcdctl to the case statement?